web-interface/controlling-vms-modules/migrate-vm-web.adoc

```diff
@@ -0,0 +1,12 @@
+[[migrate-vm-web]]
+== Live Migrating a Virtual Machine
```

Comment:
"Live Migrating" sounds odd to me, but if that's the phrase being used, I guess we'll go with it; otherwise, I'd suggest "Migrating a Running Virtual Machine". Similar changes would be necessary in the following section if that change is made, though.

Reply:
It is called the "live migration" so anyone looking for live migration would look for something like this, so I would stay with it.